src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -1136,43 +1138,37 @@ namespace HighTempRadiantSystem {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         Real64 ThisSurfIntensity; // temporary for W/m2 term for rad on a surface
 
+        // Using/Aliasing
+        auto &dataHBFS = state.dataHeatBalFanSys;
+
         // Initialize arrays
-        state.dataHeatBalFanSys->SumConvHTRadSys = 0.0;
-        state.dataHeatBalFanSys->SumLatentHTRadSys = 0.0;
-        state.dataHeatBalFanSys->SurfQHTRadSys = 0.0;
-        state.dataHeatBalFanSys->ZoneQHTRadSysToPerson = 0.0;
+        dataHBFS->SumConvHTRadSys = 0.0;
+        dataHBFS->SumLatentHTRadSys = 0.0;
+        dataHBFS->SurfQHTRadSys = 0.0;
+        dataHBFS->ZoneQHTRadSysToPerson = 0.0;
 
         for (int RadSysNum = 1; RadSysNum <= state.dataHighTempRadSys->NumOfHighTempRadSys; ++RadSysNum) {
```

Comment:
Another opportunity for object loop. You cleaned this up so much it stands out now. Now you get the point of what to look for. I am not by any means the expert but this is what I look for during code review.